agreement_project/views/agreement_view.xml

```diff
@@ -5,7 +5,7 @@
         <field name="name">Tasks</field>
         <field name="type">ir.actions.act_window</field>
         <field name="res_model">project.task</field>
-        <field name="view_type">form</field>
+        <field name="binding_view_types">form</field>
```

Comment:
I think we dont require binding_view_types please check and remove it.

Reply:
@dsolanki-initos I have done this can you please review again.